### ISSUE TITLE
Extract tokenizer information to another paper

### ISSUE THREAD

**Issue by AmitMY** [sign/image-latent-transformer#18]
Possible title: `Back to basics: Using Pure UTF-8 for Tokenization`

And explain how we can leverage control tokens, and what are pitfalls (whitespaces for example).
Cover C0, and C1.

Give notes on how to express these tokens visually in papers (visual control tokens)

**Comment by AmitMY** [comment]
From my one experiment, with a single seed, on a tiny model: https://github.com/sign/image-latent-transformer/tree/main/training/experiments/language-modelling-bit

https://wandb.ai/amit_my/clm-bit-embeddings/workspace?

### Training loss is lower for most if not all training steps

<img width="363" height="270" alt="Image" src="https://github.com/user-attachments/assets/7d1995c1-97ef-4b3e-8dda-f88461dea928" />

### Eval loss and eval accuracy is strictly better for all eval steps

<img width="743" height="288" alt="Image" src="https://github.com/user-attachments/assets/4603a1db-0fb2-433e-a13d-7ee031c64385" />

### Runtime is a bit slower

<img width="1121" height="283" alt="Image" src="https://github.com/user-attachments/assets/12459ca7-0bee-4638-9f79-ade0fc0d15d2" />

**Comment by AmitMY** [comment]
Done! https://github.com/sign/utf8-tokenizer
